### PR TITLE
Fixed _check_required_channels error

### PR DESCRIPTION
**Pull request by stephenlee91**
- if statement was missing a _
- _check_required_channels function second for loop caused an error. was checking for the name of the channel but should've been checking for the channel ID. also incorrect variable name.

### CODE REVIEW

File: cogs/moveusers.py [hunk 1]
```diff
@@ -26,12 +26,12 @@ def _find_similar_channel(channel_name, voice_channels, threshold=0.5):
...
 
     # Helper function to check if all required channels for automove are fulfilled
-    def _check_required_channels(self, ctx):
+    def _check_required_channels(self):
         for req_channel in ["Lounge", "Study", "General", "Cinema"]:
             if req_channel not in DIFFERENT_NAME_CHANNELS:
                 return False
-        for channel_name in DIFFERENT_NAME_CHANNELS.values():
-            channel = discord.utils.get(ctx.guild.voice_channels, name=channel_name)
+        for channel_id in DIFFERENT_NAME_CHANNELS.values():
```
Comment (TimothyYLin) [inline cogs/moveusers.py:33]: good catch

File: cogs/moveusers.py [hunk 2]
```diff
@@ -54,6 +54,19 @@ async def move(self, ctx, member: discord.Member, channel: str):
     # Command function that auto moves users to various different channels depending on activities of all users in all voice channels
     @commands.command()
     async def automove(self, ctx):
+        if not self._check_required_channels():
+            try:
```
Comment (TimothyYLin) [inline cogs/moveusers.py:58]: Wasn't this block already merged?

**Comment by TimothyYLin** [comment]
Most likely going to have a merge conflict after merging master into this branch.

**Review by TimothyYLin — APPROVED** [review]
Approved but need to watch out for merge conflict. Most likely going to need a merge conflict fix/rebase.

**Review by asturrus — APPROVED** [review]

**Pull request merged**